### PR TITLE
Update configure-client-of-bonita-bpm-engine.md

### PR DESCRIPTION
**Pull request by HappyBonita**
fix(doc):  Unconsistent documentation to configure Client Connection between doc pages and javadoc

Fix params for create connection

Closes [BS-15921](https://bonitasoft.atlassian.net/browse/BS-15921)

### CODE REVIEW

**Comment by redboul** [comment]
Hello,

the thing we would need to check with this issue is to understand why the use of extend names of the properties is invalid inside the engine...

thx anyway

**Comment by laurentleseigneur** [comment]
retest this please

File: md/configure-client-of-bonita-bpm-engine.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB
  * `HTTP`:
    The `HTTP` connection mode will connect the client to the engine using the http protocol. This is the most common way to connect to engine remotly.  
    When using `HTTP` mode, you must set the following system properties
-   * `org.bonitasoft.engine.api-type.server.url`
+   * `server.url`
```
Comment (educhastenier) [inline md/configure-client-of-bonita-bpm-engine.md]: When using System properties, you MUST leave the prefix `org.bonitasoft.engine.api-type.`
Please fix this.

File: md/configure-client-of-bonita-bpm-engine.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ You can set the system property `org.bonitasoft.engine.api-type` to `HTTP`, `EJB
...
       It is the url of the server, e.g. for a engine on the same host it can be `http://localhost:8080`
-   * `org.bonitasoft.engine.api-type.application.name`
+   * `application.name`
 
       It is the the of the web aplication on wich the engine HTTP API is deployed, e.g. `bonita`
```
Comment (educhastenier) [inline md/configure-client-of-bonita-bpm-engine.md]: Missing word `name` here

**Pull request merged**